docs/guides/manage-cost.mdx

```diff
@@ -125,18 +123,13 @@ No further jobs can be submitted by using this instance until the cost limit is
 
 ## How to see what you’re being charged
 
-You are sent a monthly invoice that provides details about your resource charges. You can check how much has been spent at any time on the [IBM Cloud Billing and usage page](https://cloud.ibm.com/billing).
+You or your account owner will receive a monthly invoice that provides details about your resource charges. You can check how much has been spent at any time on the [IBM Cloud Billing and usage page](https://cloud.ibm.com/billing).
```

Comment:
Is "or your account owner" true in this statement (also in line 158)? @fvarchon @Matt-Stypulkoski